src/main/resources/logback.xml

```diff
@@ -1,9 +1,12 @@
 <configuration>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <!-- encoders are assigned the type
-             ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
-        <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        <encoder class="net.logstash.logback.encoder.LogstashEncoder">
+            <includeCallerData>true</includeCallerData>
+            <prefix class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+                <layout class="ch.qos.logback.classic.PatternLayout">
+                    <pattern>%-125(%d{yyyy-MM-dd'T'HH:mm:ss.SSSZ} %X{X-Request-Id} %-5level %msg) %ex</pattern>
```

Comment:
Here I'm using the prefix to add the human readable message before the JSON object. In the future if we want only JSON to be printed, we just have to remove the prefix layout.